tools/ports/cocos2d.py

```diff
@@ -0,0 +1,130 @@
+import os
+import shutil
+import logging
+import re
+
+TAG = '-master'
```

Comment:
in the other ports we don't include the `-` in TAG, for consistency I think it's best to do the same here